bluesky_queueserver/manager/manager.py

```diff
@@ -944,29 +944,29 @@ async def _queue_item_move_handler(self, request):
         """
         logger.info("Removing item from the queue.")
         try:
-            plan, qsize, msg = {}, None, ""
+            item, qsize, msg = {}, None, ""
             pos = request.get("pos", None)
             uid = request.get("uid", None)
             pos_dest = request.get("pos_dest", None)
             before_uid = request.get("before_uid", None)
             after_uid = request.get("after_uid", None)
-            plan, qsize = await self._plan_queue.move_item(
+            item, qsize = await self._plan_queue.move_item(
                 pos=pos, uid=uid, pos_dest=pos_dest, before_uid=before_uid, after_uid=after_uid
             )
             success = True
         except Exception as ex:
             success = False
             msg = f"Failed to move the item: {str(ex)}"
 
-        return {"success": success, "msg": msg, "plan": plan, "qsize": qsize}
+        return {"success": success, "msg": msg, "item": item, "qsize": qsize}
 
     async def _queue_clear_handler(self, request):
         """
         Remove all entries from the plan queue (does not affect currently executed run)
         """
         logger.info("Clearing the queue")
         await self._plan_queue.clear_queue()
-        return {"success": True, "msg": "Plan queue is now empty."}
+        return {"success": True, "msg": ""}
```

Comment:
When is the `msg` supposed to be non-empty?

Reply:
`msg` contains error message if request fails. For the requests that never fail `msg` is always empty string. The idea is that implementation of the API may change or API functionality may be expanded, so that there will be failing cases.

Reply:
Over DM, @dmgav explained that if the `success=True`, then `msg` should always be empty.